cmd/cdi/cmd/validate.go

```diff
@@ -49,6 +50,15 @@ were reported by the cache.`,
 				fmt.Printf("  %2d: %v\n", idx, strings.TrimSpace(err.Error()))
 			}
 		}
+
+		for _, v := range cache.ListVendors() {
+			for _, s := range cache.GetVendorSpecs(v) {
```

Comment:
I can see this nested loop in other places as well. Would it make sense to wrap it in function?